container/webui/conf/openqa.ini

```diff
@@ -1,5 +1,5 @@
 [auth]
-method = OpenID
+method = Fake
```

Comment:
This is not something we should introduce into our production templates of the container setup. Otherwise people might risk spawning an openQA instance with no proper authentication

Reply:
Done